### PR TITLE
[DW-2030] Events page broken

### PR DESCRIPTION
**Pull request by John-Milne**

### CODE REVIEW

File: site/components/src/main/java/uk/nhs/digital/common/components/FeedListComponent.java [hunk 1]
```diff
@@ -180,5 +188,21 @@ public void doBeforeRender(HstRequest request,
...
+
+    @Override
+    public Filter createQueryFilter(HstRequest request, HstQuery hstQuery) throws FilterException {
+        return super.createQueryFilter(request, hstQuery);
+    }
+
+    @Override
+    public void applyAndFilters(HstQuery hstQuery, List<BaseFilter> filters) throws FilterException {
+        super.applyAndFilters(hstQuery, filters);
+    }
```
Comment (Deven-Rawat) [inline site/components/src/main/java/uk/nhs/digital/common/components/FeedListComponent.java:206]: Do we need to create above methods here?
getComponentParametersInfo
createQueryFilter
applyAndFilters

File: site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java [hunk 2]
```diff
@@ -31,6 +31,7 @@
 public class EventsComponent extends EssentialsEventsComponent {
 
     private static Logger log = LoggerFactory.getLogger(EventsComponent.class);
+    ComponentHelper componentHelper = new ComponentHelper();
```
Comment (Deven-Rawat) [inline site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java:34]: private?

File: site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java [hunk 3]
```diff
@@ -114,4 +120,20 @@ protected int getPageSize(final HstRequest request, final EssentialsPageable par
         return NumberUtils.isCreatable(compononentPageSize)
             ? Integer.parseInt(compononentPageSize) : super.getPageSize(request, paramInfo);
     }
+
```
Comment (Deven-Rawat) [inline site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java:123]: Do we need to create above methods here?
getComponentParametersInfo
createQueryFilter
applyAndFilters

File: site/components/src/main/java/uk/nhs/digital/common/components/FeedListComponent.java (hunk 1 above)
Comment (John-Milne) [reply site/components/src/main/java/uk/nhs/digital/common/components/FeedListComponent.java:206]: Yes, as the super methods have protected access modifiers and thus are not accessible outside of their class or sub-class

File: site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java (hunk 3 above)
Comment (John-Milne) [reply site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java:123]: Yes, as the super methods have protected access modifiers and thus are not accessible outside of their class or sub-class

**Pull request merged**